tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Update repository cache (apt)
+  apt:
+    update_cache: yes
+  changed_when: false
+  when: "ansible_pkg_mgr == 'apt'"
+
+- name: Update repository cache (pacman)
+  pacman:
+    update_cache: yes
+  changed_when: false
+  when: "ansible_pkg_mgr == 'pacman'"
+
+- name: Install smartmontools (apt)
+  apt:
```

Comment:
Maybe use "package" instead of "apt" and pacman"? Might cover more distributions

Reply:
I think it's obvious that smartmontools is a package and it is being installed using apt (in this case).

Reply:
You didn't get my point, but it doesn't matter